src/test-cases-app/test-cases-app.js

```diff
@@ -2,11 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import ReactTestKitchen from '@mapbox/react-test-kitchen';
 import componentIndex from './component-index'; // eslint-disable-line
+import { UserContextProvider } from '../components/page-layout/context/user-context';
 
 class App extends React.Component {
   render() {
     return (
-      <ReactTestKitchen projectTitle="dr-ui" componentIndex={componentIndex} />
+      <UserContextProvider>
```

Comment:
UserContextProvider will be wrapped inside PageLayout. We need to add it here to hook update the context for the examples.